etc/formatter/format.sh

```diff
@@ -54,39 +54,48 @@ else
     printHelp
 fi
 
+declare -a PIDS=()
+
 # As specified in `docker-compose.formatter.yml`
 readonly repo_root="/workdir"
 
-echo "--- Prettier Typescript"
 prettier \
     --config prettierrc.toml \
     ${prettier_arg} \
     ${repo_root}/src/js/graphql_endpoint/**/*.ts \
     ${repo_root}/src/js/engagement_view/src/**/*.ts \
-    ${repo_root}/src/js/engagement_view/src/**/*.tsx
+    ${repo_root}/src/js/engagement_view/src/**/*.tsx &
+PIDS+=("$!")
 
 # Slightly different config for yaml
-echo "--- Prettier YAML"
 prettier \
     --config prettierrc-yaml.toml \
     ${prettier_arg} \
     ${repo_root}/**/*.yml \
     ${repo_root}/**/*.yaml \
     ${repo_root}/.buildkite/**/*.yml \
-    ${repo_root}/.github/**/*.yml
+    ${repo_root}/.github/**/*.yml &
```

Comment:
Does Prettier ignore dot files in its globs? I'm just wondering why the Buildkite and Github files have to be called out specifically here.

Reply:
I believe so, yes.